course/assignments/reproducible-research-assignment.md

```diff
@@ -0,0 +1,21 @@
+## Reproducible Research Plan:
```

Comment:
Overall I like this. When was this assignment going to be given? It might be better to give this out on Oct. 17 when projects are started, since we can get them to complete this assignment after being introduced to common elements of an R project. That way we get them to stay on top of their project while also not having to take time away from the lecture trying to help them get their project details laid out.